change/@microsoft-teams-js-f6daee5c-d395-44ed-a4f3-670c3bbc44fe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added private DataLayer capability",
+  "packageName": "@microsoft/teams-js",
+  "email": "kerrynb@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
i assume it is okay to mark as patch even though it is a breaking change since this is internal API?

Reply:
It isn't a patch, it is minor bump (the top type is the type to look at here). Trevor said that since these APIs are marked beta the breaking changes are not an issue (this would be true even if they weren't private I assume).

`"type": "minor",`

Reply:
Correct, if something is marked as beta we reserve the right to change the signature with little or no notice.